website/docusaurus.config.js

```diff
@@ -0,0 +1,185 @@
+  "tagline": "Apache APISIX is a dynamic, real-time, high-performance Cloud-Native API gateway, based on the Nginx library and etcd.",
+  "url": "https://apisix.apache.org/",
+  "baseUrl": "/",
+  "organizationName": "Apache",
+  "projectName": "apisix-website",
+  "scripts": [
+    "https://buttons.github.io/buttons.js"
+  ],
+  "favicon": "img/favicon.png",
+  "customFields": {
+    "tagline2": "Apache APISIX software provides rich traffic management features such as load balancing, dynamic upstream, canary release, circuit breaking, authentication, observability, and more.",
+    "users": [
+      {
+        "caption": "360",
+        "image": "https://static.apiseven.com/2020/05/WechatIMG618.png",
+        "infoLink": "https://www.360.com",
+        "pinned": true
+      },
+      {
+        "caption": "HelloTalk",
+        "image": "https://static.apiseven.com/2020/05/HelloTalk.png",
+        "infoLink": "https://www.hellotalk.com/?lang=en",
+        "pinned": true
+      },
+      {
+        "caption": "网易",
+        "image": "https://static.apiseven.com/2020/05/1588907762-WechatIMG2916.png",
+        "infoLink": "https://www.163.com",
+        "pinned": true
+      },
+      {
+        "caption": "腾讯云",
+        "image": "https://static.apiseven.com/2020/05/%E8%85%BE%E8%AE%AF%E4%BA%91-1536x546.jpg",
+        "infoLink": "https://qcloud.com",
+        "pinned": true
+      },
+      {
+        "caption": "中国航信",
+        "image": "https://static.apiseven.com/2020/05/%E4%B8%AD%E5%9B%BD%E8%88%AA%E4%BF%A1.png",
+        "infoLink": "http://www.infosky.com.cn/publish/main/index.html",
+        "pinned": true
+      }
+    ],
+    "repoUrl": "https://github.com/apache/apisix",
+    "docsUrl": ""
+  },
+  "onBrokenLinks": "log",
+  "onBrokenMarkdownLinks": "log",
+  "presets": [
+    [
+      "@docusaurus/preset-classic",
+      {
+        "docs": {
+          "showLastUpdateAuthor": true,
+          "showLastUpdateTime": true,
+          "path": "../docs",
+          "sidebarPath": "../website/sidebars.json"
+        },
+        "blog": {
+          "path": "blog"
+        },
+        "theme": {
+          "customCss": "../src/css/customTheme.css"
+        }
+      }
+    ]
+  ],
+  "plugins": [],
+  "themeConfig": {
+    "navbar": {
+      "title": "Apache APISIX™",
+      "style": 'primary',
+      "logo": {
+        "src": "img/logo.png"
+      },
+      "items": [
+        {
+          "to": "docs/",
+          "label": "Docs",
+          "position": "left"
+        },
+        {
+          "to": "/blog",
+          "label": "Blog",
+          "position": "left"
+        },
+        {
+          "to": "docs/downloads",
+          "label": "Downloads",
+          "position": "left"
+        },
+        {
+          "to": "docs/team",
+          "label": "Team",
+          "position": "left"
+        },
+        {
+          "to": "/help",
+          "label": "Help",
+          "position": "left"
+        }
+      ]
```

Comment:
nav should be right in the v1 version

Reply:
It doesn't have to be IMHO, keep the original V2 style is ok.

Reply:
Sorry, it’s my fault, I didn’t make it clear, I actually want to ask if we need to be consistent with V1.

Reply:
no no no, don't have to haha